### ISSUE TITLE
Karma segmentation fault on Linux 

### ISSUE THREAD

**Issue by pfps** [usc-isi-i2/Web-Karma#416]
On one of my Linux machines Karma immediately throws a segmentation fault on startup.

The machine where the problem occurs is running kernel 5.0.7 and has openjdk 1.8.0_201.
The machine where everything is fine is running kernel 4.15.0 and has openjdk 11.0.2.

Any ideas on what could be causing this?

**Comment by pfps** [comment]
The segmentation fault persists even if I update java to the same version on the machine that is working.

**Comment by ldibanyez** [comment]
Hi,
Same issue here on a Debian 10 VM (VirtualBox) with kernel 4.19.37-5. I tried with both OpenJDK 11 and Oracle SE 11.

A valgrind trace revealed an issue with the libnode.so. Apparently some apps that ship their own libnode.so like Slack and Skype sometimes have this issue. Following what worked for Slack [here](http://ubuntuhandbook.org/index.php/2018/11/slack-app-not-launching-ubuntu-18-10/), I installed libnode-dev and replaced the shipped libnode.so with the newer one. 

Unfortunately, I ran into:
undefined symbol: _ZN4node20g_upstream_node_modeE

Which in the context of other apps means that Karma cannot work with a newer libnode.so (10.15.2~dfsg-2 in my case).

I gave up and compiled it from source. It worked.
